### PR TITLE
IBP-6352-fix-changeset-with-column-as-null

### PR DESCRIPTION
**Pull request by cuenyad**
Please, review

### CODE REVIEW

File: src/main/resources/liquibase/crop_changelog/18_4_0.xml [hunk 1]
```diff
@@ -73,8 +73,32 @@
 			</column>
 		</createTable>
 	</changeSet>
+	<changeSet author="vanina" id="v18.4.0-4-fix">
```
Comment (clarysabel) [inline src/main/resources/liquibase/crop_changelog/18_4_0.xml:76]: @cuenyad modify author

File: src/main/resources/liquibase/crop_changelog/18_4_0.xml [hunk 2]
```diff
@@ -73,8 +73,32 @@
...
+					foreign key (cvterm_id) references cvterm (cvterm_id),
+				constraint fk_file_metadata_cvterm_file_metadata_file_id
+					foreign key (file_metadata_id) references file_metadata (file_id)
+			)
+				comment 'cvterms to formally describe an image or file';
 
-	<changeSet author="nahuel" id="v18.4.0-4" >
+		</sql>
+	</changeSet>
+	<!-- Commented changeSet for issue found it when create a Primary Key allowing null columns
```
Comment (clarysabel) [inline src/main/resources/liquibase/crop_changelog/18_4_0.xml:99]: @cuenyad after tested, removed commented code

File: src/main/resources/liquibase/workbench_changelog/12_5_0.xml [hunk 3]
```diff
@@ -424,8 +424,36 @@
 			ON UPDATE NO ACTION;
 		</sql>
 	</changeSet>
-
-	<changeSet author="vanina" id="v12.5.0-17">
+	<changeSet author="vanina" id="v12.5.0-17-fix">
```
Comment (clarysabel) [inline src/main/resources/liquibase/workbench_changelog/12_5_0.xml:427]: @cuenyad modify author

File: src/main/resources/liquibase/workbench_changelog/12_5_0.xml [hunk 4]
```diff
@@ -424,8 +424,36 @@
...
+					ON DELETE NO ACTION
+					ON UPDATE NO ACTION,
+				CONSTRAINT fk_role_permission_2
+					FOREIGN KEY (permission_id)
+				REFERENCES permission (permission_id)
+					ON DELETE NO ACTION
+					ON UPDATE NO ACTION);
+		</sql>
+	</changeSet>
+	<!-- Commented changeSet for issue found it when create a Primary Key allowing null columns
```
Comment (clarysabel) [inline src/main/resources/liquibase/workbench_changelog/12_5_0.xml:454]: @cuenyad remove commented code after tested

File: src/main/resources/liquibase/crop_changelog/18_4_0.xml (hunk 1 above)
Comment (cuenyad) [reply src/main/resources/liquibase/crop_changelog/18_4_0.xml]: done

File: src/main/resources/liquibase/crop_changelog/18_4_0.xml (hunk 2 above)
Comment (cuenyad) [reply src/main/resources/liquibase/crop_changelog/18_4_0.xml]: done

File: src/main/resources/liquibase/workbench_changelog/12_5_0.xml (hunk 4 above)
Comment (cuenyad) [reply src/main/resources/liquibase/workbench_changelog/12_5_0.xml]: done

File: src/main/resources/liquibase/workbench_changelog/12_5_0.xml (hunk 3 above)
Comment (cuenyad) [reply src/main/resources/liquibase/workbench_changelog/12_5_0.xml]: done

**Pull request merged**